### ISSUE TITLE
Unable to Verify sessions from Android only

### ISSUE THREAD

**Issue by ItsRogueRen** [commetchat/commet#380]
**Describe the bug**
When trying to verify sessions, verification from the Android app never prompts anything on the desktop side. Verifying the Android app from the desktop app does work as expected.

**To Reproduce**
1. Download Commet on Android
2. Download any desktop client for Matrix, Commet included. 
3. On the Android app, go to settings>security and verify a desktop session
4. verification prompt will never appear on the desktop side. Tested with Commet, Cinny, and Element on desktop. 

**Expected behavior**
A prompt to appear on the desktop side to verify the session.

**Screenshots**

**Additional context**

<details open>
<summary>Device Information</summary>
<br>

**Device**
Platform: `android`
Version: `v0.3.0`
Git Hash: `0fc02086a3d641496eb01943ae0c68a97c0b277d`
Detail: `default`

**System Info**

Product: `genevn_g`
</details>

**Comment by Airyzz** [comment]
Hi, I am unable to reproduce this error. Is there any information that could be helpful in logs?

**Comment by Erquint** [comment]
Don't have logs on me, but concur that I had the same issue on separate Android devices.  
Think I only ever managed to confirm a single session of a single account in Commet if any.

**Comment by watchdominiondotcom** [comment]
yeah i have the same issue and i cant delete any logins or devices. i havent tried on web though. all others i have.

**Comment by Da0neDatGotAway** [comment]
Same issue +1 Using element mobile app and matrix.org homeserver
